bloom-housing/bloom
# feat: retract modal

This PR addresses #4188


## Description

Hooks up the retract modal in the lottery flow.

## How Can This Be Tested/Reviewed?

On a closed listing with a lottery as an admin, run and release a lottery. When released, the only action available to you should be a new retract button. Ensure the modal matches the [copy](https://docs.google.com/document/d/130OuxoQnKnukoVr_SPQq57A0JZ4vycdcS9q0EX2Gkso/edit#heading=h.gxdoimgubxbk), and that retracting it sends it back to the run state.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
